Cargo.toml

```diff
@@ -24,9 +24,11 @@ keywords = ["cql2"]
 
 [dependencies]
 boon = "0.6.0"
+derive_is_enum_variant = "0.1.1"
```

Comment:
FWIW I use https://crates.io/crates/enum-as-inner for this functionality. It looks to be much more popular and more recently maintained.